delta-api/src/main/java/io/cdap/delta/api/Configurer.java

```diff
@@ -18,9 +18,19 @@
 
 import io.cdap.cdap.api.plugin.PluginConfigurer;
 
+import javax.annotation.Nullable;
+
 /**
  * Configures a data transfer.
  */
 public interface Configurer extends PluginConfigurer {
 
+  @Nullable
```

Comment:
Mainly because I don't see a way to get the namespace and name here:
https://github.com/data-integrations/delta/blob/develop/delta-app/src/main/java/io/cdap/delta/app/DeltaApp.java#L50